src/main/java/io/mosip/digitalcard/config/SwaggerConfig.java

```diff
@@ -59,43 +59,43 @@ public class SwaggerConfig {
 	 * 
 	 * @return {@link ApiInfo}
 	 */
-	private ApiInfo apiInfo() {
-		return new ApiInfoBuilder().title(TITLE).description(DISCRIBTION).version(DIGITALCARD_SERVICE_VERSION).build();
-	}
+//	private ApiInfo apiInfo() {
+//		return new ApiInfoBuilder().title(TITLE).description(DISCRIBTION).version(DIGITALCARD_SERVICE_VERSION).build();
+//	}
 
 	/**
 	 * Produce Docket bean
 	 * 
 	 * @return Docket bean
 	 */
-	@Bean
-	public Docket api() {
-		boolean swaggerBaseUrlSet = false;
-		if (!localEnv && swaggerBaseUrl != null && !swaggerBaseUrl.isEmpty()) {
-			try {
-				proto = new URL(swaggerBaseUrl).getProtocol();
-				host = new URL(swaggerBaseUrl).getHost();
-				port = new URL(swaggerBaseUrl).getPort();
-				if (port == -1) {
-					hostWithPort = host;
-				} else {
-					hostWithPort = host + ":" + port;
-				}
-				swaggerBaseUrlSet = true;
-			} catch (MalformedURLException e) {
-				
-			}
-		}
-
-		Docket docket = new Docket(DocumentationType.SWAGGER_2).apiInfo(apiInfo())
-				.groupName(TITLE).select().apis(RequestHandlerSelectors.any())
-				.paths(PathSelectors.regex("(?!/(error).*).*")).build();
-		if (swaggerBaseUrlSet) {
-			docket.protocols(protocols()).host(hostWithPort);
-		}
-
-		return docket;
-	}
+//	@Bean
```

Comment:
Why is this commented? Please fix the swagger also.

Reply:
We are not using swagger for the digital-card because the other repo which are dependent on these repo will take care of that.
Also the digital-card is using the older version of swagger.